packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePluginTest.kt

```diff
@@ -225,30 +246,315 @@ class AmplifyDataStorePluginTest {
                     dataStoreItemChange)
             null
         }.`when`(mockAmplifyDataStorePlugin).observe(any<Consumer<Cancelable>>(),
-                                                     any<Consumer<DataStoreItemChange<out Model>>>(),
-                                                     any<Consumer<DataStoreException>>(),
-                                                     any<Action>())
+                any<Consumer<DataStoreItemChange<out Model>>>(),
+                any<Consumer<DataStoreException>>(),
+                any<Action>())
 
         flutterPlugin.onSetupObserve(mockResult)
 
         verify(mockResult, times(1)).success(true)
         verify(mockStreamHandler, times(1)).sendEvent(eventData)
     }
 
+    @Test
+    fun test_hub_outboxMutationEnqueued_event() {
```

Comment:
This file is growing out of control, can we move the hub events test to a separate test file?

Reply:
Fixed.